contribs/gmf/apps/appmodule.js

```diff
@@ -15,4 +15,4 @@ goog.require('gmf');
 /**
  * @type {!angular.Module}
  */
-var appModule = angular.module('app', [gmfModule.name]);
+var appModule = angular.module('app', [gmfModule.name, 'ngTouch']);
```

Comment:
You're probably right. But this means that we will have to load `ngTouch` in every single example or app in `gmf`.
In my opinion, the `ngTouch` module should only be required for the mobile app.

By the way, the desktop application is broken since dependency on `ngTouch` is not satisfied.